frontend/assets/stylesheets/components/nav/_nav.scss

```diff
@@ -274,6 +222,55 @@ $global-nav-height: 36px;
 }
 
 /* ==========================================================================
+   Secondary Navigation
+   ========================================================================== */
+
+.nav-secondary {
```

Comment:
Trying to move away from using modifiers for the navigation components, as they don't actually share a huge number of traits so are not really modifiers. Feels clearer and more predictable as separate components.